scipy/stats/_continuous_distns.py

```diff
@@ -2224,20 +2227,21 @@ def _entropy(self, a, c):
 
 
 class genhalflogistic_gen(rv_continuous):
-    """A generalized half-logistic continuous random variable.
+    r"""A generalized half-logistic continuous random variable.
 
     %(before_notes)s
 
     Notes
     -----
-    The probability density function for `genhalflogistic` is::
+    The probability density function for `genhalflogistic` is:
+
+    .. math::
 
-        genhalflogistic.pdf(x, c) =
-            2 * (1-c*x)**(1/c-1) / (1+(1-c*x)**(1/c))**2
+        f(x, c) = \frac{2 {(1-c x)}^{\frac{1}{c-1}}}{{\left (1+{(1-c x)}^{1/c} \right)}^2}
```

Comment:
Better (APS) latex styling, and easier to read as text:
`\frac{2 (1 - c x)^{1/(c-1)}}{[1 + (1 - c x)^{1/c}]^2}`